hibernate-core/src/main/java/org/hibernate/cache/spi/CacheTransactionSynchronization.java

```diff
@@ -48,8 +48,10 @@ public interface CacheTransactionSynchronization {
 	 *
 	 * @implSpec This "timestamp" need not be related to timestamp in the Java
 	 * Date/millisecond sense.  It just needs to be an incrementing value.
+	 *
+	 * An UnsupportedOperationException is thrown for No Caching
```

Comment:
I'm not sure if it's clear what "No Caching" means. Could you clarify that?

Reply:
changed, what do you think?